code/game/jobs/whitelist.dm

```diff
@@ -35,7 +35,7 @@ GLOBAL_LIST_EMPTY(whitelist)
 				list("ckey" = M.ckey)
 			)
 
-			if(!job_read.warn_execute())
+			if(!job_read.warn_execute(async = FALSE)) // Dont async this. Youll make roundstart slow.
```

Comment:
At some point ill likely refactor these to use a GLOB and have them all cache in at world init with a single query that can run async, instead of 300 queries running async.